Azure/azure-cli
# [AppService] az webapp up/create: better error handling of app name with period

**Description**
Fixes #16622 
Run `az webapp up/create` with a name that has a period (".")
![image](https://user-images.githubusercontent.com/14339314/105387061-06c49500-5bca-11eb-8a2b-dd2eeaf6f252.png)

Made this fix on CLI instead of the checkNameAvailability API because "." is valid for a full site hostname (URL's allow periods), just not valid for an appservice name. 
I think portal also does some frontend validations like this when creating a webapp (though I haven't confirmed this)

**Testing Guide**
Expected msg: 
![image](https://user-images.githubusercontent.com/14339314/105387131-18a63800-5bca-11eb-967b-99e595a08901.png)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
